_i18n/nl.yml

```diff
@@ -0,0 +1,314 @@
+  aregvrtanesyan:
+    name: Areg Vrtanesyan
+    title: Onderzoeker/ontwikkelaar cryptografie
+    bio: Areg is een cryptografie-ingenieur in Firo en studeert momenteel Cybersecurity aan YSU. Heeft ervaring in software engineering en back-end programmeren in C++. Zijn huidige interesses zijn cryptografie, informatiebeveiliging en muziek.
+  tigranmamikonyan:
+    name: Tigran Mamikonyan
+    title: Research
+    bio: Tigran heeft een master in informatica en toegepaste wiskunde en computer- en informatiewetenschappen. Momenteel promoveert hij bij de Faculteit Informatica en Toegepaste Wiskunde aan de Staatsuniversiteit van Yerevan.
+  
+roadmap:
+  aura: Aura Private Voting (finalizing paper)
+  auradesc: Aura is ons private stemmechanisme voor DAO's, geïnspireerd op onze Lelantus-technologie die anoniem maar controleerbaar stemmen mogelijk maakt en ook de voortgang van de stemming verbergt. We zijn van plan om dit te gebruiken als basis voor de ontwikkeling naar geformaliseerd gedecentraliseerd bestuur.
+  avalanche: Avalanche
+  avalanchedesc: Avalanche is een consensusalgoritme dat directe transacties, verbeterde beveiliging en fork-free upgrades mogelijk maakt. We onderzoeken Avalanche-consensus als een mogelijke vervanging voor onze masternodes.
+  development: Ontwikkeling
+  heading: Roadmap
+  mnodes: Deterministische Masternodes (gedaan)
+  mnodesq: Long Lived Masternode Quorums  (gedaan)
+  mtpstrip: MTP-strip (gedaan)
+  mtpstripdesc: MTP-strippen reduceert de grootte van Firo's blokketen van 60GB naar 4GB door MTP-bewijzen van oudere blokken te strippen.
+  rebrand: Rebrand  (gedaan)
+  lelantus: Lelantus-activatie  (gedaan)
+  lelantusdesc: Lelantus is ons privacyprotocol van de volgende generatie dat de hoogste niveaus van praktische privacy biedt zonder vertrouwde opstelling en alleen vertrouwend op solide cryptografische aannames. Lelantus ondersteunt anonimiteitssets die vele malen groter zijn dan andere privacysystemen zonder vertrouwde opstelling.
+  chainlocks: Chainlocks  (gedaan)
+  chainlocksdesc: Chainlocks maken gebruik van Firo's masternodenetwerk om bescherming te bieden tegen 51%-aanvallen. Transacties worden definitief zodra 1 blok is bevestigd.
+  rap: RAP-adressen  (gedaan)
+  rapdesc: RAP-adressen staan een adres toe dat publiekelijk kan worden gedeeld, zodat derden niet kunnen zien hoeveel of wanneer het geld is ontvangen.
+  instant: Instant Send  (gedaan)
+  instantdesc: Instant Send maakt gebruik van Firo's masternodenetwerk om transacties binnen een paar seconden te bevestigen, waardoor het praktisch kan worden gebruikt als ruilmiddel.
+  elysium: Elysium (on Testnet)
+  elysiumdesc: Elysium maakt het mogelijk om tokens (bijvoorbeeld stablecoins, asset backed coins) uit te geven op het netwerk van Firo die profiteren van de privacytechnologie van Firo.

```

Comment:
```suggestion
  elysiumdesc: Elysium maakt het mogelijk om tokens (bijvoorbeeld stablecoins, asset backed coins) uit te geven op het netwerk van Firo die profiteren van de privacy technologie van Firo.

```

Reply:
Sorry, the words need to be joined in Dutch concatenation of nouns, so without a space.